src/stores/my-account.ts

```diff
@@ -58,6 +82,10 @@ export const followedIdsStorage = new LocalStorageAndForage(
 )
 export const hasSentMessageStorage = new LocalStorage(() => 'has-sent-message')
 const accountStorage = new LocalStorage(() => ACCOUNT_STORAGE_KEY)
+const parentProxyAddressStorage = new LocalStorage(
+  () => CONNECTED_WALLET_ADDRESS_STORAGE_KEY
+)
+const preferredWalletStorage = new LocalStorage(() => 'preferred-wallet')
```

Comment:
You use const for all, but here use hardcoded value

Reply:
I removed all constants and use it directly inside localstorage